src/en/howto-privatecloud.md

```diff
@@ -264,7 +264,7 @@ Now generate relevant metadata for the tools by running the command:
 
     juju metadata generate-tools -d <tools_dir>
 
-Finally, the contents of  can be uploaded to a location in the Juju metadata
+Finally, the contents of <tools_dir> can be uploaded to a location in the Juju metadata
```

Comment:
This won't work. It will be interpreted as HTML. Use a tick to wrap as code blocks